src/components/PageHeader/PageHeader.tsx

```diff
@@ -89,7 +98,7 @@ export const PageHeader = ({
       <div className={styles['page-header__left']}>
         {kicker && <p className={styles['page-header__kicker']}>{kicker}</p>}
```

Comment:
Let's use the `Text` component here and point it to the `overline` size.

Reply:
Wondering if this should be a more generic slot with this to put badges and stuff in there